src/vcpkg/commands.build.cpp

```diff
@@ -778,6 +780,29 @@ namespace vcpkg
             variables.emplace_back("ARIA2", paths.get_tool_exe(Tools::ARIA2, stdout_sink));
         }
 
+        if (!args.cmake_debug.empty())
+        {
+            if (args.cmake_debug.size() == 1 ||
```

Comment:
This seems to suggest that it should be a single option rather than a multi option? (Does it make sense to pass `--x-cmake-debug` multiple times?)

Reply:
I see now, you're abusing that multi-options separate the ;s for you. I don't think that's right though, since it allows multiple uses of that switch when only one should be accepted.

What do you think about https://github.com/BillyONeal/vcpkg-tool/commit/7dcd7924f7f46d414e54215b8a8ea6fc722745c8 ?

Reply:
I just made that as a PR: https://github.com/Neumann-A/vcpkg-tool/pull/1

Reply:
> I see now, you're abusing that multi-options separate the ;s for you. I don't think that's right though, since it allows multiple uses of that switch when only one should be accepted.

Yes. Simply because I didn't knew better and couldn't see through what the parser was doing for asset&binary options. I simply couldn't see the constructor of  `Something(StringView setting);` for custom types.